client/src/app/site/pages/meetings/pages/motions/components/motion-forward-dialog/components/motion-forward-dialog/motion-forward-dialog.component.html

```diff
@@ -18,6 +18,10 @@ <h4>{{ committee.name }}</h4>
                         }}"
                     >
                         {{ meeting.name }}
+                            &nbsp;&middot;&nbsp;
+                            {{ meeting.start_time | localizedDate: 'll' }}
+                            &nbsp;-&nbsp;
+                            {{ meeting.end_time | localizedDate: 'll' }}
```

Comment:
What happens if start/end time is null? You probably need to wrap it in some container